### ISSUE TITLE
Cannot resolve SentryModule as an NgModule

### ISSUE THREAD

**Issue by jeremypele** [FinanzRitter/nativescript-sentry#2]
There's an issue with the plugin module compilation for Angular.

### Which platform(s) does your issue occur on?
- Any platform/device

### Please, provide the following version numbers that your issue occurs with:
```
ERROR in node_modules/nativescript-sentry/angular/app.module.d.ts:7:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

This likely means that the library (nativescript-sentry/angular) which declares SentryModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

7 export declare class SentryModule {
```

**Comment by jeremypele** [comment]
Got the same issue on my app and on the plugin demo FYI

Edit: forgot to mention that it's happening when building with Ivy

**Comment by bradmartin** [comment]
Fairly certain that the plugin needs to be updated with how it builds the ngModule using ng-packagr.  The NS plugins monorepo has this working OOTB with all the plugins. I don't know angular that well or the dist system with ng-packagr but I have worked on the plugins monorepo and Nathan set it up so that it's very simple to add ngModules to any plugin and the build scripts package and distribute the modules correctly. So might be worth pulling some scripts from there  https://github.com/NativeScript/plugins/blob/b9b6559aac06cbcb0ed2b104dcb2cbc4b44ca336/tools/scripts/build-finish.ts#L17
https://github.com/NativeScript/plugins

**Comment by jeremypele** [comment]
Thanks @bradmartin , I'll have a look at this and try to solve this issue

**Comment by juniorschen** [comment]
@jeremypele 
Some forecast for merge?

**Comment by jeremypele** [comment]
@juniorschen Hi, there's an opened PR https://github.com/FinanzRitter/nativescript-sentry/pull/3 but I don't have any rights to merge it.
In the meantime, we're using the package I've build on the PR fork. Feel free to use it (https://github.com/jeremypele/nativescript-sentry/releases/tag/2.0.0)

**Comment by mm-psymbl** [comment]
@jeremypele any idea on the merge?  how do i use your package? 
How do i reference your forked version in my package.json?

**Comment by mwagnurr** [comment]
@mm-psymbl you could download the package directly and install it from the file, or host it on your own npm registry (if applicable), or maybe there is also a way to install it from Github directly somehow.

@sc85 when can we expect the open PR to be reviewed and merged?  
On our side the PR changes fixed our issues and the plugin works again.

**Comment by sc85** [comment]
Sorry for the detail. I must have missed any notification emails.
I just released v2.0.1. To get the Actions working again I had to update some dependencies which hopefully didn't break anything else.

Please give at try and let me know if it works for you.
@jeremypele Thanks PR

**Comment by mwagnurr** [comment]
Thanks a lot! So far it is working good for us 👍
